src/main/java/org/assertj/core/api/AbstractFileAssert.java

```diff
@@ -691,6 +692,21 @@ public SELF hasName(String expected) {
     return myself;
   }
 
+  /**
+   * Alias for {@link #hasName(String)}.
+   *
+   * @param expected the expected {@code File} name.
+   * @return {@code this} assertion object.
+   * @throws NullPointerException if the expected name is {@code null}.
+   * @throws AssertionError if the actual {@code File} is {@code null}.
+   * @throws AssertionError if the actual {@code File} does not have the expected name.
+   *
+   * @see java.io.File#getName() name definition.
+   */
```

Comment:
I'd rather reuse the javadoc of `hasName` and introduce `@since`. Something like:

```suggestion
  /**
   * Verifies that the actual {@code File} has given name. Alias for {@link #hasName(String)}.
   *
   * <p>
   * Example:
   * <pre><code class='java'> File xFile = new File(&quot;somewhere/xFile.java&quot;);
   * File xDirectory = new File(&quot;somewhere/xDirectory&quot;);
   *
   * // assertion will pass
   * assertThat(xFile).hasFileName(&quot;xFile.java&quot;);
   * assertThat(xDirectory).hasFileName(&quot;xDirectory&quot;);
   *
   * // assertion will fail
   * assertThat(xFile).hasFileName(&quot;xFile&quot;);
   * assertThat(xDirectory).hasFileName(&quot;somewhere&quot;);</code></pre>
   *
   * @param expected the expected {@code File} name.
   * @return {@code this} assertion object.
   * @throws NullPointerException if the expected name is {@code null}.
   * @throws AssertionError if the actual {@code File} is {@code null}.
   * @throws AssertionError if the actual {@code File} does not have the expected name.
   *
   * @see java.io.File#getName() name definition.
   * @see #hasName(String)
   * @since 3.20.0
   */
```

Please also add:

```java
   * @see #hasFileName(String)
```

to the javadoc of `hasName`.

Reply:
Thank for the inputs, javadoc updated 👍